src/app/docs/confirm/confirm-docs.component.html

```diff
@@ -0,0 +1,11 @@
+<sky-docs-demo-page
+  moduleName="SkyConfirmModule"
+  moduleSourceCodePath="src/app/public/modules/confirm/"
+  pageTitle="Confirm"
+>
+  <sky-docs-demo-page-summary>
+    The confirm service launches simple confirmation dialogs to allow users to confirm actions. The <sky-code>SkyConfirmService</sky-code> provider launches the confirmation dialog. Within the service, you can specify the dialog's message and customize the button text.
```

Comment:
I'd stop the summary after the first sentence to follow the shorter style we've adopted. I think we can copy the rest (starting with `Launches...`) to a description for the `SkyConfirmService`

Reply:
Done